docs/_static/custom.js

```diff
@@ -33,4 +33,24 @@ $(document).ready(function () {
     $("option[value='" + selected + "']").attr("selected", "selected");
     window.location = document.URL.replace(current, selected);
   });
+
+  // Cookbook Onboard (AI Assistant). 
+  // API key is public so it's fine to just hardcode it here.
+  var COOKBOOK_PUBLIC_API_KEY = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiI2NmFhY2QzYjZiYmI2MmQwOGY0ZjYzNzgiLCJpYXQiOjE3MjI0Njk2OTEsImV4cCI6MjAzODA0NTY5MX0.Xw4JO3X3T19NOGqPnsz_DYDEBdsxNYM9JkZK4k8ADs8";
```

Comment:
What is the point of a public API key?

Reply:
It is used to make user interactions with Ask Cookbook available for your review in the CMS, allowing you to further tune replies by leaving feedback on conversations. Additionally, it enables you to dynamically configure the appearance of Ask Cookbook using the same CMS